app/models/message_draft.rb

```diff
@@ -0,0 +1,5 @@
+# frozen_string_literal: true
+
+# Message drafts are essentially messages with an implied folder (drafts)
+class MessageDraft < Message
+end
```

Comment:
But actually, I don't think MessageDraft needs validations, Message does, right? You could put the validations in a common concern and mix it in as appropriate.